packages/nimble-components/src/table-column/date-text/cell-view/index.ts

```diff
@@ -20,22 +21,23 @@ export class TableColumnDateTextCellView extends TableColumnTextCellViewBase<
 TableColumnDateTextCellRecord,
 TableColumnDateTextColumnConfig
 > {
-    private static readonly formatter = new Intl.DateTimeFormat(undefined, {
-        dateStyle: 'medium',
-        timeStyle: 'medium'
-    });
+    private formatter?: Intl.DateTimeFormat;
+
+    private columnConfigChanged(): void {
+        this.formatter = createFormatter(this.columnConfig);
```

Comment:
Now that I'm seeing what it looks like for everything to have its own formatter (cell view, column, and group header view), I'm wondering if it would be possible to avoid recreating formatters so often (particularly since the cell view will exist for every single row in the table).

Would it be possible to put the formatter in the columnConfig so that each cell view and each header view are all referencing the instance owned by the column itself?

Reply:
I talk about this in a response to an earlier (unresolved) comment. Let's continue the discussion in that thread: https://github.com/ni/nimble/pull/1381#discussion_r1272773193